### ISSUE TITLE
E0120 09:32:45.604616 1854 model_repository_manager.cc:1002] Poll failed for model directory 'ensemble_model': output 'decoded_sequence' for ensemble 'ensemble_model' is not written

### ISSUE THREAD

**Issue by legerez** [triton-inference-server/server#5268]
Description
I have created an ensemble model that has a preprocessor script that is on python backend  as first step and a postprocessor script that is also on python backend as a last step. I keep getting this error message which I don't really understand. Could someone help me figure it out ?

Triton Information
TRITON_IMAGE = 'nvcr.io/nvidia/tritonserver:22.11-py3'

Are you using the Triton container or did you build it yourself?
Triton container

To Reproduce

the config.pbtxt for the ensemble model
`name: "ensemble_model"
platform: "ensemble"

input [
  {
    name: "image"
    data_type: TYPE_UINT8
    dims: [-1]
  }
]
output [
  {
    name: "decoded_sequence"
    data_type: TYPE_STRING	
    dims: [-1]
  }
]

ensemble_scheduling {
  step [
    {
      model_name: "preprocess"
      model_version: 1
      input_map {
        key: "image"
        value: "image"
      }
      output_map {
        key: "pixel_values"
        value: "pixel_values"
      }
    },
    {
      model_name: "encoder"
      model_version: 1
      input_map {
        key: "pixel_values"
        value: "pixel_values"
      }
      output_map {
            key: "last_hidden_state"
            value: "last_hidden_state"
        }
      output_map {
            key: "17338"
            value: "17338"
        }
      
    },
    {
      model_name: "decoder"
      model_version: 1
      input_map {
            key: "input_ids"
            value: "input_ids"
        }
      input_map {
            key: "attention_mask"
            value: "attention_mask"
        }
      input_map {
            key: "encoder_hidden_states"
            value: "last_hidden_state"
        }
      output_map {
        key: "logits"
        value: "logits"
        }
      output_map {
        key: "onnx::Reshape_283"
        value: "onnx::Reshape_283"
        }
      output_map {
        key: "onnx::Reshape_296"
        value: "onnx::Reshape_296"
        }
      output_map {
        key: "onnx::Reshape_404"
        value: "onnx::Reshape_404"
        }
      output_map {
        key: "onnx::Reshape_417"
        value: "onnx::Reshape_417"
        }
      output_map {
        key: "onnx::Reshape_533"
        value: "onnx::Reshape_533"
        }
      output_map {
        key: "onnx::Reshape_546"
        value: "onnx::Reshape_546"
        }
      output_map {
        key: "onnx::Reshape_654"
        value: "onnx::Reshape_654"
        }
      output_map {
        key: "onnx::Reshape_667"
        value: "onnx::Reshape_667"
        }
      output_map {
        key: "onnx::Reshape_783"
        value: "onnx::Reshape_783"
        }
      output_map {
        key: "onnx::Reshape_796"
        value: "onnx::Reshape_796"
        }
      output_map {
        key: "onnx::Reshape_904"
        value: "onnx::Reshape_904"
        }
      output_map {
        key: "onnx::Reshape_917"
        value: "onnx::Reshape_917"
        }
      output_map {
        key: "onnx::Reshape_1033"
        value: "onnx::Reshape_1033"
        }
      output_map {
        key: "onnx::Reshape_1046"
        value: "onnx::Reshape_1046"
        }
      output_map {
        key: "onnx::Reshape_1154"
        value: "onnx::Reshape_1154"
        }
      output_map {
        key: "onnx::Reshape_1167"
        value: "onnx::Reshape_1167"
        }
    },
    {
      model_name: "postprocess"
      model_version: 1
      input_map {
        key: "logits"
        value: "logits"
      }
      output_map {
        key: "decoded_sequence"
        value: "decoded_sequence"
      }
    }
  ]
}`

the config.pbtxt for postprocessor:
`name: "postprocess"
backend: "python"

input [
  {
    name: "logits"
    data_type: TYPE_FP32
    dims: [-1,-1,57582]
  }
]
output [
  {
    name: "decoded_sequence"
    data_type: TYPE_STRING	
    dims: [-1]
  }
]

instance_group [
  {
    kind: KIND_GPU
  }
]`

**Comment by xanderdunn** [comment]
@legerez What was the solution here?

**Comment by legerez** [comment]
@xanderdunn There was a problem with the input maps in the decoder when fixed got rid of the error.
